src/extensions/savetoLaserfiche/SaveToLaserficheDialog.tsx

```diff
@@ -57,6 +57,8 @@ function SaveToLaserficheDialog(props: {
     NgElement & WithProperties<LfLoginComponent>
   >();
 
+  const spDevMode = window?.localStorage.getItem(SPDEVMODE_LOCAL_STORAGE_KEY);
+  const region = spDevMode && spDevMode === 'true' ? 'a.clouddev.laserfiche.com' : 'laserfiche.com';

```

Comment:
where do we set devMode in localStorage?

Reply:
we don't, it will be a manual step.. should be added in the docs somewhere, but probably not publicly..